switchapi/src/switch_packet.c

```diff
@@ -156,41 +186,198 @@ switch_packet_rx_from_hw()
 }
 
 void
-switch_packet_tx_to_host(switch_hostif_info_t *hostif_info, char *packet, int packet_size)
+switch_packet_tx_to_host(
+        switch_packet_header_t *packet_header,
+        char *packet,
+        int packet_size)
 {
-    int intf_fd = hostif_info->intf_fd;
-    if (write(intf_fd, packet, packet_size) < 0) {
+    static char                       in_packet[SWITCH_PACKET_MAX_BUFFER_SIZE];
+    switch_cpu_header_t               *cpu_header = NULL;
+    switch_packet_rx_entry_t          rx_entry;
+    switch_packet_rx_info_t           *rx_info = NULL;
```

Comment:
I was confused to see rx_info, rx_entry being used inside "packet_tx_to..." function. But looks like tx_to_host really means packet received from the network and going up the host stack?
Typically all functions in this path should be called "_rx_.."?